tests/neo4j/txrun.py

```diff
@@ -40,3 +38,100 @@ def test_does_not_update_last_bookmark_on_rollback(self):
         tx.rollback()
         bookmarks = self._session.lastBookmarks()
         self.assertEqual(len(bookmarks), 0)
+
+    def test_does_not_update_last_bookmark_on_failure(self):
+        session = self._driver.session("w")
+        tx = session.beginTransaction()
+        with self.assertRaises(types.responses.DriverError):
+            tx.run("RETURN").next()
+            tx.commit()
+        bookmarks = session.lastBookmarks()
+        self.assertEqual(len(bookmarks), 0)
+
+    def test_should_be_able_to_rollback_a_failure(self):
+        if get_driver_name() not in ["go"]:
```

Comment:
```suggestion
        if get_driver_name() in ["go"]:
```
Or is GO really the only one that works?

Reply:
typo man strikes again.